docs/deploying-airbyte/on-restack.md

```diff
@@ -0,0 +1,70 @@
+# On Restack
+
+## Overview
+Scaling Airbyte on Kubernetes has never been easier. <br/>[Restack](https://www.restack.io) is a platform that lets you discover and deploy open source tools on Kubernetes. With just a few clicks, you can deploy and scale Airbyte in your own cloud infrastructure on kubernetes so that developers can concentrate on what they do best and don’t have to worry about devops.
+
+## Getting Started
+
+You can start your Airbyte journey with Restack with just a few steps.
+
+  - [Sign up](#sign-up)
+  - [Add AWS credentials with AdministratorAccess](#add-aws-credentials-with-administratoraccess)
+  - [One-click cluster creation with Restack](#one-click-cluster-creation-with-restack)
+  - [Deploy Airbyte on Restack](#deploy-airbyte-on-restack)
+  - [Start using Airbyte](#start-using-airbyte)
+  - [Deploy multiple instances of Airbyte](#deploy-multiple-instances-of-airbyte)
+
+## Sign up
+
+To Sign-up to Restack, visit [www.restack.io/signup](https://www.restack.io/signup). You can sign up with your corporate email address or your GitHub profile. You do not need a credit card to sign up.
+If you already have an account, go ahead and login to Restack at [www.restack.io/login](https://www.restack.io/login)
+
+## Add AWS credentials with AdministratorAccess
+
+Restack’s architecture is designed in such a way, that your products like Airbyte and any data stored is only located in your cloud. The Restack control plane only accesses DevOps related metrics and logs, to maintain Airbyte in your cloud.
+
+To deploy Airbyte in your own AWS infrastructure with Restack, you will need to add your credentials as the next step. 
+
+Please make sure that this account has `AdministratorAccess`. This is how Restack can ensure an end-to-end cluster creation and cluster management process.
+
+1. Navigate to `Clusters` in the left-hand navigation menu.
```

Comment:
Please follow our style guide (all UI elements are bold and not in code blocks. Replace `text` with **text** wherever applicable)